packages/@aws-cdk/app-delivery/test/test.pipeline-deploy-stack-action.ts

```diff
@@ -8,6 +8,7 @@ import * as iam from '@aws-cdk/aws-iam';
 import * as s3 from '@aws-cdk/aws-s3';
 import * as cxschema from '@aws-cdk/cloud-assembly-schema';
 import * as cdk from '@aws-cdk/core';
+import * as constructs from 'constructs';
```

Comment:
From a style POV, I would prefer this to be `import { Construct } from 'constructs';` no?

Reply:
That's ok, not that important.